### PR TITLE
E2E tests on jobs page

### PR DESCRIPTION
**Pull request by ayuwono**
- Tests on jobs page (export, delete, facets, search, output)
- Add verification for input and harmonize flows on jobs and dashboard page
- Add some page objects

### CODE REVIEW

File: quick-start/e2e/specs/jobs/jobs.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+      it ('check and export some jobs', function() {
+        jobsPage.jobCheckboxByPosition(5).click();
+        jobsPage.jobCheckboxByPosition(3).click();
+        jobsPage.actionDropDown().click();
+        browser.wait(EC.elementToBeClickable(jobsPage.exportActionMenuItem()));
+        jobsPage.exportActionMenuItem().click();
+        browser.wait(EC.elementToBeClickable(jobsPage.exportButton()));
+        jobsPage.exportButton().click();
+        browser.sleep(5000);
```
[the remark this answers is not in the document]
Comment (ayuwono) [reply quick-start/e2e/specs/jobs/jobs.ts]: Yes, this sleep is a trial, there is no snack bar component for the download/export status. I can remove this

[the remark this answers is not in the document]
Comment (ayuwono) [reply quick-start/e2e/specs/jobs/jobs.ts]: I might be able to use this: https://stackoverflow.com/questions/41082777/protractor-test-download-file-without-knowing-filename

But maybe for the next time

[the remark this answers is not in the document]
Comment (aebadirad) [reply quick-start/e2e/specs/jobs/jobs.ts]: Sounds good.

File: quick-start/e2e/specs/jobs/jobs.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import { protractor , browser, element, by, By, $, $$, ExpectedConditions as EC} from 'protractor';
```
Comment (dmcassel) [inline quick-start/e2e/specs/jobs/jobs.ts]: IntelliJ says there are several unused imports. 

- protractor, element, by, By, $, $$ from protractor
- import { pages } from '../../page-objects/page';
- import loginPage from '../../page-objects/auth/login';
- import entityPage from '../../page-objects/entities/entities';
- import browsePage from '../../page-objects/browse/browse';
- import viewerPage from '../../page-objects/viewer/viewer';

**Comment by dmcassel** [comment]
There are currently failing tests, but @ayuwono understands the problem and should be pushing a fix soon.

**Pull request merged**